osutil/overlay_linux.go

```diff
@@ -69,6 +69,16 @@ func IsRootWritableOverlay() (string, error) {
 					continue
 				}
 
+				if dir == "/media/root-rw/overlay" {
+					// On the Ubuntu server ephemeral image, '/' is setup via
+					// overlayroot (on at least 18.10), which uses a combination
+					// of overlayfs and chroot. This differs from the livecd setup
```

Comment:
no chroot us used. pivot is done into the rootfs from the initramfs, but that is the common path of initramfs.